src/main/resources/db/changelog/changes/v1.0.0/sql/2021-07-01-01__create_itm_contrib_opt_conf_table.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE item_contribution_options_configuration
+(
+    central_server_id UUID NOT NULL PRIMARY KEY,
+    created_by VARCHAR(255),
+    created_date TIMESTAMP,
+    last_modified_by VARCHAR(255),
+    last_modified_date TIMESTAMP,
+    update_counter INTEGER,
+    CONSTRAINT fk_item_contribution_options_configuration_central_server FOREIGN KEY (central_server_id) REFERENCES central_server (id)
```

Comment:
`ON DELETE CASCADE`

`central_server_id` here must be not null, so if deletion on cascade is not enabled then removal of central server that has related item contribution config will fail.